develop-docs/sdk/telemetry/logs.mdx

```diff
@@ -268,33 +268,39 @@ An SDK should implement [Tracing without Performance](/sdk/telemetry/traces/trac
 
 ### Default Attributes
 
-By default the SDK should set the following attributes:
+By default the SDK should attach the following attributes to a log:
 
-1. `sentry.message.template`: The parameterized template string
-2. `sentry.message.parameter.X`: The parameters to the template string. X can either be the number that represent the parameter's position in the template string (`sentry.message.parameter.0`, `sentry.message.parameter.1`, etc) or the parameter's name (`sentry.message.parameter.item_id`, `sentry.message.parameter.user_id`, etc)
-3. `sentry.environment`: The environment set in the SDK
-4. `sentry.release`: The release set in the SDK
-5. `sentry.trace.parent_span_id`: The span id of the span that was active when the log was collected. This should not be set if there was no active span.
-6. `sentry.sdk.name`: The name of the SDK that sent the log
-7. `sentry.sdk.version`: The version of the SDK that sent the log
-8. [BACKEND SDKS ONLY] `server.address`: The address of the server that sent the log. Equivalent to `server_name` we attach to errors and transactions.
-
-Example:
+1. `sentry.environment`: The environment set in the SDK if defined.
+2. `sentry.release`: The release set in the SDK if defined.
+3. `sentry.trace.parent_span_id`: The span id of the span that was active when the log was collected. This should not be set if there was no active span.
+4. `sentry.sdk.name`: The name of the SDK that sent the log
+5. `sentry.sdk.version`: The version of the SDK that sent the log
```

Comment:
Do we also plan to support integrations, features, and settings that [already exist](https://develop.sentry.dev/sdk/data-model/event-payloads/sdk/)? These can be handy for investigating issues and tracking the adoption of integrations and features.

Reply:
At the current moment no. We want to avoid making the payload too large.

We can re-evaluate this based on internal and external feedback.